src/Core/Prefab/GlobeView.js

```diff
@@ -194,6 +194,8 @@ function GlobeView(viewerDiv, coordCarto, options = {}) {
     // Setup View
     View.call(this, 'EPSG:4978', viewerDiv, options);
 
+    options.object3d = options.object3d || this.scene;
```

Comment:
Same question: why did you change this?

Reply:
I don't think you need to modify GlobeView, except for adding the `setZFactor` method.